system/classes/route.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * route class which separates the controller, methods and parameters from
+ * the url
+ * 
+ */
+
+class route 
+{
+  /**
+   * 
+   * @var $controller: stores the controller name
+   * it initially consists of default value home
+   * @var $method: stores the method name
+   * it initially consists of default value index
+   * @var $param: stores parameters that are passed in the url
+   * 
+   */
+
+  protected $controller = "home";
+  protected $method = "index";
+  protected $param = [];
+  
+  /**
+   * 
+   * constructor
+   * This separates the controller name, method name and parameters and stores
+   * in the variables, after that the callback function is used.
+   * 
+   */
```

Comment:
Please maintain the coding standards when you write down the comments for a function.